_licenses/ecl-2.0.txt

```diff
@@ -0,0 +1,238 @@
+---
+title: Educational Community License v2.0
+spdx-id: ECL-2.0
+source: https://opensource.org/licenses/ECL-2.0
+
+description: The Educational Community License version 2.0 ("ECL") consists of the Apache 2.0 license, modified to change the scope of the patent grant in section 3 to be specific to the needs of the education communities using this license.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+note: The Apereo Foundation recommends taking the additional step of adding a boilerplate notice to the header of each source file. You can find the notice at the very end of the license in the appendix.
+
+using:
+  - Sakai: https://github.com/sakaiproject/sakai/blob/master/LICENSE
+  - OAE: https://github.com/oaeproject/Hilary/blob/master/LICENSE
+  - Opencast Matterhorn: https://bitbucket.org/opencast-community/matterhorn/src/e5b1684a822c0bd9bb07f3e53b4e1f22932da5ef/LICENSE?fileviewer=file-view-default
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - patent-use
+  - private-use
+
+conditions:
+  - include-copyright
+  - document-changes
+
+limitations:
+  - trademark-use
+  - liability
+  - warranty
+
+---
+Educational Community License, Version 2.0 (ECL-2.0)
```

Comment:
I don't think this line is supposed to be part of the license text; it's fully redundant with the next lines and not included in https://github.com/spdx/license-list/blob/788e54b566dc78358e3e1c3f9b7187ad4005114e/ECL-2.0.txt